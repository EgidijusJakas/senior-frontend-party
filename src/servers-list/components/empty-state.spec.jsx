import React from 'react';
import { shallow } from 'enzyme';
import EmptyState from './empty-state';

describe('Empty State', () => {
    it('should should render empty state', () => {
        const renderedComponent = shallow(<EmptyState />);
        expect(renderedComponent).toMatchSnapshot();
    });
});
