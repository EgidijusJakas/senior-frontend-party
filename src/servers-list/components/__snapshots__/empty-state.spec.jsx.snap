// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty State should should render empty state 1`] = `
<Styled(styled.li)
  className="list-group-item"
>
  <span>
    No Results
  </span>
</Styled(styled.li)>
`;
